ts/store/reducers/__mock__/idps.ts

```diff
@@ -0,0 +1,89 @@
+import { SpidIdp } from "../../../../definitions/content/SpidIdp";
+
+// since this is a test SPID idp, we set isTestIdp flag to avoid rendering.
+// It is used has a placeholder to handle taps count on it and open when
+// taps count threadshold is reached (see https://www.pivotaltracker.com/story/show/172082895)
+export const testIdp: SpidIdp = {
+  id: "test",
+  name: "Test",
+  logo:
+    "https://raw.githubusercontent.com/pagopa/io-services-metadata/master/spid/idps/spid.png",
+  entityID: "test-login",
+  profileUrl: "",
+  isTestIdp: true
+};
+
+export const idps: ReadonlyArray<SpidIdp> = [
+  {
+    id: "arubaid",
+    name: "Aruba",
+    logo:
+      "https://raw.githubusercontent.com/pagopa/io-services-metadata/master/spid/idps/spid-idp-arubaid.png",
+    entityID: "arubaid",
+    profileUrl: "http://selfcarespid.aruba.it"
```

Comment:
Not strictly related with this pr because the datamodel is the same of the previous local format but as general discussion, we need both `id` and `entityID` in our datamodel? Seems that are always the same, maybe we can remove one field or they have different meaning?